modules/reference/pages/properties/topic-properties.adoc

```diff
@@ -508,6 +508,28 @@ CAUTION: Setting `redpanda.remote.readreplica` together with either `redpanda.re
 
 Configure Redpanda-specific topic properties.
 
+[[redpandaleaderspreference]]
+==== redpanda.leaders.preference
+
+The preferred location (rack) for partition leaders of a topic.
+
+This property inherits the value from the config_ref:default_leaders_preference,true,properties/cluster-properties[] cluster configuration property. You may override the cluster-wide setting by specifying the value for individual topics.
+
+If the cluster configuration property config_ref:enable_rack_awareness,true,properties/cluster-properties[] is set to `false`, leader pinning is disabled across the cluster.
+
+**Default**: `none`
+
+**Values**:
+
+- `none`: Opt out the topic from leader pinning.
+- `racks:<rack1>[,<rack2>,...]`: Preferred location (rack) of all topic partition leaders to brokers with these rack IDs. The list can contain one or more rack IDs. 
```

Comment:
> Preferred location (rack) of all topic partition leaders to brokers with these rack IDs

The phrasing looks a bit awkward here as there is no verb. What about just "Preferred location (racks) of all topic partition leaders"?